data/main_menu.json

```diff
@@ -8,7 +8,7 @@
         "discord_invite_code" : "v4PU3WeH7z"
     },
 
-    "prank_sound": false,
+    "prank_sound": true,
```

Comment:
🤨

Reply:
Huh why not, tomorrow is the day

Reply:
Better early then late, we do have the date check itself